src/stage1/codegen.cpp

```diff
@@ -565,7 +565,7 @@ static LLVMValueRef make_fn_llvm_value(CodeGen *g, ZigFn *fn) {
 
         bool want_ssp_attrs = g->build_mode != BuildModeFastRelease &&
                               g->build_mode != BuildModeSmallRelease &&
-                              g->link_libc;
+                              g->link_libc && !target_is_wasm(g->zig_target);
```

Comment:
note: this is necessary to avoid stack protector insertion even with link_libc =true for Wasm target.

Reply:
Wouldn't a comment explaining that be necessary?

Reply:
https://github.com/ziglang/zig/pull/9227/commits/fd0effffc29bae7ed7491c72365b279ec3da145f done!